Makefile

```diff
@@ -7,7 +7,7 @@ PS2PDF = ps2pdf
 CHEAT_NAME = ROScheatsheet
 
 PDF_FILES = $(subst dia,pdf,$(shell ls images/*.dia))
-YOUR_LOGO = your-logo
+YOUR_LOGO = logo_tork_vertical_sized_g-apps
```

Comment:
> Not confident how this line works.

https://github.com/tork-a/cheatsheet/pull/2/files#diff-b67911656ef5d18c4ae36cb6741b7965R22 
line cecks if YOUR_LOGO exists and use sed to fix tex files

--
◉ Kei Okada


On Mon, May 18, 2015 at 8:51 AM, Isaac I.Y. Saito <notifications@github.com> 
wrote:

> In Makefile 
> <https://github.com/tork-a/cheatsheet/pull/2#discussion_r30476406>:
>
> > @@ -7,7 +7,7 @@ PS2PDF = ps2pdf
> >  CHEAT_NAME = ROScheatsheet
> >  
> >  PDF_FILES = $(subst dia,pdf,$(shell ls images/*.dia))
> > -YOUR_LOGO = your-logo
> > +YOUR_LOGO = logo_tork_vertical_sized_g-apps
>
> Not confident how this line works.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/tork-a/cheatsheet/pull/2/files#r30476406>.
>